Makefile

```diff
@@ -9,7 +9,7 @@ test:
 phan:
 	$(DC_RUN_PHP) env PHAN_DISABLE_XDEBUG_WARN=1 php ./vendor/bin/phan
 psalm:
-	$(DC_RUN_PHP) php ./vendor/bin/psalm --threads=1
+	$(DC_RUN_PHP) php ./vendor/bin/psalm --threads=1 --no-cache
```

Comment:
I'm stumped by this - it should be really easy, and I wonder if it's my version of make (`GNU Make 4.2.1`), but nproc works locally, and in the project's image, but not via make.
I went back through history and found that threads was changed to 1 in #409 recently, so it might be safest to not play with it anyway!!

Reply:
Ok, no problem, no need to spend to much time on it for this PR. Thanks anyway!